policy-service/src/component-tests/java/uk/gov/gchq/palisade/component/policy/service/RedisPolicyCachingTest.java

```diff
@@ -57,18 +49,16 @@
 import static org.junit.jupiter.api.Assertions.assertAll;
 
 @SpringBootTest(
-        classes = {ApplicationConfiguration.class, CacheAutoConfiguration.class},
+        classes = {ApplicationConfiguration.class, DefaultConfiguration.class, CacheAutoConfiguration.class},
         webEnvironment = WebEnvironment.NONE,
-        properties = {"spring.cache.redis.timeToLive=1s"}
+        properties = {"spring.cache.redis.timeToLive=1s", "spring.cache.redis.keyPrefix=test"}
```

Comment:
Prefer something closer to reality (considering redis keyspace separators etc.)
```suggestion
        properties = {"spring.cache.redis.timeToLive=1s", "spring.cache.redis.keyPrefix=test:"}
```

Reply:
Done